src/engine/dataapi/DataStream.java

```diff
@@ -8,7 +8,7 @@
 /**
  * Data stream passed to and from the controller and data storage.
  */
-public class DataStream implements ProtoDataStream, Serializable{
```

Comment:
This is too funny. The way she emphasized to never use serializable.

Reply:
lol